dnf/util.py

```diff
@@ -643,30 +643,16 @@ def _is_file_pattern_present(specs):
     return False
 
 
-def is_container():
+def is_bootc_host():
     """Returns true is the system is managed as an immutable container,
        false otherwise.  If msg is True, a warning message is displayed
        for the user.
     """
+    ostree_booted = '/run/ostree-booted'
```

Comment:
Minor/optional: feels worth hoisting this up to a constant at the top of the file.

Reply:
I don't see much constants in the file, maybe we can do this when we need it for other functions?

Reply:
I like the variables defined for readability, but either way is fine by me.